vsphere-mixin/panels.libsonnet

```diff
@@ -0,0 +1,1472 @@
+            },
+          },
+          {
+            id: 'filterFieldsByName',
+            options: {
+              include: {
+                names: [
+                  'vcenter_datacenter_name 1',
+                  'vcenter_datastore_name',
+                  'Value #disk_total',
+                  'Value #disk_utilization',
+                  'Value #disk_available',
+                ],
+              },
+            },
+          },
+          {
+            id: 'organize',
+            options: {
+              excludeByName: {},
+              includeByName: {},
+              indexByName: {
+                'Value #disk_total': 2,
+                'Value #disk_utilization': 3,
+                'Value #disk_available': 4,
+                'vcenter_datacenter_name 1': 0,
+                vcenter_datastore_name: 1,
+              },
+              renameByName: {
+                'Value #disk_total': 'Disk total',
+                'Value #disk_utilization': 'Disk utilization',
+                'Value #disk_available': 'Disk free',
+                'vcenter_datacenter_name 1': 'Datacenter',
+                vcenter_datastore_name: 'Datastore',
+              },
+            },
+          },
+        ]),
+
+      topCPUUsageResourcePools:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top CPU usage by resource pools',
+          targets=[t.topCPUUsageResourcePools],
+          description='The resource pools with the highest CPU usage in the datacenter.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('rotmhz'),
+
+      topMemoryUsageResourcePools:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top memory usage by resource pools',
+          targets=[t.topMemoryUsageResourcePools],
+          description='The resource pools with the highest memory usage in the datacenter.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('mbytes'),
+
+      topCPUShareResourcePools:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top CPU shares by resource pools',
+          targets=[t.topCPUShareResourcePools],
+          description='The resource pools with the highest amount of CPU shares allocated in the datacenter.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('shares'),
+
+      topMemoryShareResourcePools:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top memory shares by resource pools',
+          targets=[t.topMemoryShareResourcePools],
+          description='The resource pools with the highest amount of memory shares allocated in the datacenter.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('shares'),
```

Comment:
These four I'd like to see increased heights for, as table-based legend makes sense. 
It **might** fit as right-aligned, but if not then I think 25% taller would fix it.

e.g. either of the lefthand options
![image](https://github.com/grafana/jsonnet-libs/assets/1282515/30c868fb-533b-4d74-850c-a174a9c61189)